jenkins/ci.suse.de/templates/cloud-socok8s-integration.yaml

```diff
@@ -91,8 +91,8 @@
       - choice:
           name: deployment
           choices:
-            - 'airship'
             - 'osh'
+            - 'airship'
           default: 'osh'
```

Comment:
Please remove this line as it does nothing. But may confuse someone.

Reply:
Done